styleguide/components/21-atoms/buttons/_buttons.scss

```diff
@@ -52,16 +52,26 @@
   border-radius: $radius-1;
   outline: 0;
   font-family: $default-font-family;
+  font-weight: 600;
+  text-align: center;
 }
 
 //
-// Makes a button block instead of inline.
+// Makes a button display block instead of inline.
 //
 @mixin button-block {
   display: block;
   width: 100%;
 }
 
+//
+// Make a button display inline instead of block.
+//
+@mixin button-inlineblock {
```

Comment:
Aren't buttons inline by default?
So either this is redundant, or change the comment to 'inline-block instead if inline'.

Reply:
This is more of a fallback function for overriding other css when necessary. (just in case...).
But I'll change the comment to inline-block.

Reply:
Remove the 'instead of block' and we have a deal!